deck/views.py

```diff
@@ -26,6 +26,10 @@ def success_redirect(self, message):
         messages.success(self.request, message)
         return HttpResponseRedirect(self.get_success_url())
 
```

Comment:
There should have two lines between the classes.

You can see all this in the PEP8: https://www.python.org/dev/peps/pep-0008/